content/en/docs/tasks/policy-enforcement/control-headers/index.md

```diff
@@ -1,10 +1,17 @@
 ---
-title: Control Headers and Routing
+title: Control Headers and Routing (Deprecated)
 description: Shows how to modify request headers and routing using policy adapters.
 weight: 20
 keywords: [policies,routing]
 ---
 
+{{< warning >}}
+The mixer policy is deprecated in Istio 1.5 and not recommended for production usage.
+
+Consider using Envoy `Ext-Auth`, `lua` filter, or write a filter using the
```

Comment:
Link to https://www.envoyproxy.io/docs/envoy/v1.13.0/intro/arch_overview/security/ext_authz_filter